History.md

```diff
@@ -1,5 +1,10 @@
 ## v.NEXT
 
+* `observe`/`observeChanges` callbacks are now bound.
```

Comment:
(This would go into effect only if you have not bound your function yourself to some other `this`.)